apps/organization/filters.py

```diff
@@ -32,6 +34,16 @@ def filter_organization_kinds(self, qs, name, value):
             return qs
         return qs.filter(organization_kind__in=value).distinct()
 
+    def filter_order_country_first(self, qs, name, value):
+        if not value:
+            return qs
+        country_organization_ids = qs.filter(countries__in=value).distinct().values_list('id', flat=True)
```

Comment:
This should be enough for this.
```suggestion
        country_organization_ids = qs.filter(countries__in=value).values('id').distinct()
```